litmus-portal/frontend/src/pages/HomePage/index.tsx

```diff
@@ -43,9 +41,8 @@ const HomePage = () => {
   const { userData } = useSelector((state: RootState) => state);
   const { name } = userData;
   const classes = useStyles();
-  if (userData.email === '') {
-    return <WelcomeModal isOpen />;
-  }
+  if (userData.email === '') {}
+    //return <WelcomeModal isOpen />;
```

Comment:
Can you uncomment this? Since Welcome Modal is dependent on the isOpen props

Reply:
I removed complete Welcome Modal component so that's why i uncomment that.

Reply:
/preview 10